### ISSUE TITLE
Incorrectly reports 3-phase supply

### ISSUE THREAD

**Issue by RG-129** [custom-components/zaptec#151]
Hi, firstly I'm new to HA so apologies if I'm missing something obvious. For the last 2 nights I've had the charge to my car restricted to 11A. Checking all the settings I found that the supply was being reported as 3-phase with 11A on each phase, rather than single phase at 32A. I've managed to change the available current to 32A and the car now charges OK but it does still think I have a 3-phase supply. I can't find anyway to change that. The settings I can find in the Zaptec web dashboard all look to be OK so I'm thinking it's an 
error with this integration, but maybe not. Any clues?

**Comment by sveinse** [comment]
The Zaptec integration does not control the charge current, this is handled by the Zaptec cloud and the charger itself. However, there might be a setting that's not working right for you.

What does the installation entities report? The sensors "Available current phase 1-3", "Max current" and "Network type"? And in the charger device: "Charger max current" and "Charger min current" ?

Alternatively you can download the diagnostics file and post it here. It should be sufficiently anonymized making it safe to share.

**Comment by RG-129** [comment]
Available current now shows as 32A on each phase but that's because I've manually changed the available current control from 11A. The network type is showing as TN 3 phase, which is incorrect because it should be TN 1 phase. 
Max 32A, min 6A. 
Maybe Zaptec got the settings wrong somewhere but everything on the portal seems to be set right. 
Not quite sure how to access the diagnostics file!

**Comment by sveinse** [comment]
Are the TN 1 phase correct indicated in Zaptec portal? Installations -> Circuits. Does it show "TN 1 phase, 32A" there? If it does and HA shows "TN 3 phase", then there is an issue with the integration. If not, then its incorrect configuration by your charger and you need to reach out to Zaptec on how to fix that.

Diagnostics download is described in the README here: https://github.com/custom-components/zaptec?tab=readme-ov-file#diagnostics

**Comment by RG-129** [comment]
Do jpegs display here?
Tried uploading a screenshot.
![Z TN3 2](https://github.com/user-attachments/assets/404f16b4-377a-40f6-af1e-9691cb453df2)
I think the problem must be with Zaptec. I contacted the helpdesk yesterday morning but have still heard nothing back. On several of the portal pages it mentions a 1 phase 32A supply (eg 32A @ TN φ1) but on the charging circuit page it says 3 phase. I'm guessing, therefore, that the integration is just copying Zaptec's error.
 "Id": "<--Installation[f435ab]-->",
            "Name": "<--Redact #4-->",
            "Address": "<--Redact #4-->",
            "ZipCode": "<--Redact #5-->",
            "City": "<--Redact #6-->",
            "CountryId": "f743b0d4-1f38-43e8-a95d-cf18fe1c3ede",
            "InstallationType": 1,
            "MaxCurrent": 32.0,
            "AvailableCurrent": 32.0,
            "AvailableCurrentPhase1": 32.0,
            "AvailableCurrentPhase2": 32.0,
            "AvailableCurrentPhase3": 32.0,
            "AvailableCurrentMode": 0,
            "AvailableCurrentScheduleWeekendActive": false,
            "InstallationCategoryId": "d72d6374-7f73-4df5-8056-60635b177421",
            "InstallationCategory": "Private_Installation_Category",
            "UseLoadBalancing": true,
            "IsRequiredAuthentication": true,
            "Latitude": 0.0,
            "Longitude": 0.0,
            "Active": true,
            "NetworkType": 4,
            "AvailableInternetAccessPLC": false,
            "AvailableInternetAccessWiFi": false,
            "CreatedOnDate": "2024-07-19T11:39:32.45",
            "UpdatedOn": "2024-12-07T11:13:07.457",
            "CurrentUserRoles": 3,
            "AuthenticationType": 0,
            "MessagingEnabled": true,
            "RoutingId": "default",

**Comment by sveinse** [comment]
Its possible that this is due to a misconfiguration from the installer: https://help.zaptec.com/hc/en-001/articles/13139473434513-How-to-install-Zaptec-Go#01GZK6R4E1RQQ51WAZ4BGGDB7J

As you can see here, selecting TN 1 phase should be done by the installer. Best of luck figuring it out. And if there is anything else related to the HA integration, please don't hesitate to reach out.

**Comment by RG-129** [comment]
Yeah, it's odd that it's all worked fine until a couple of days ago. Something's changed somehow. Hopefully Zaptec will get back to me.
Thanks for your input.

**Comment by sveinse** [comment]
@RG-129 did you get any more info from Zaptec about the matter. There is another issue along the same lines here as well, so I'm interested to hear what you managed to accomplish with Zaptec support.

If there is no more activity relating this issue, I propose we close this issue.
